yt/yt/tests/integration/queues/test_queue_agent.py

```diff
@@ -3995,6 +3995,37 @@ def assert_exported_table_count(count):
 
         self.remove_export_destination(export_dir)
 
+    @authors("pavel-bash")
+    @pytest.mark.timeout(150)
+    def test_cron_annotation_schedule(self):
+        # Check that the new CRON annotation works correctly, at least for this use-case.
+
+        _, queue_id = self._create_queue("//tmp/q")
+
+        export_dir = "//tmp/export"
+        self._create_export_destination(export_dir, queue_id)
+
+        export_cron_expression = "0/5 * * * * *"  # Every 5 seconds.
```

Comment:
I just realized, cron expressions with seconds are not explicitly standard. Linux cron only has minutes, I think. We should mention the fact that seconds are supported in the comments. And probably give a link to the README/header of the library we are using where the expressions are described.